internal/pkg/helm/values_inspector.go

```diff
@@ -91,3 +93,63 @@ func isBooleanYamlValue(input *map[string]interface{}, path *[]string, pathIndex
 		}
 	}
 }
+
+
+//GetValues .. Get values  fro given path
+func GetValues(arg string) (*map[string]interface{},  error) {
+
+	argString := strings.Replace(arg, ".Values.", "", -1)
```

Comment:
I think or I can give 1 as the last parameter, Since this is a path argument for values, I think there will be only one .Values.   Example.Values.image.policy

Reply:
Done